_storage/digital-ocean-deploy.md

```diff
@@ -129,7 +129,29 @@ On your local workstation, create a bucket policy to grant read permission on yo
    s3cmd setpolicy gaiahub-policy s3://meepers
    ```
 
-## Task 3: Create a DigitalOcean droplet
+## Task 3: Set a CORS configuration
+
+1. On your local workstation, create a file called `gaiahub-cors.xml` that looks like this:
+
+      ```xml
+      <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
+        <CORSRule>
+          <AllowedMethod>GET</AllowedMethod>
+          <AllowedMethod>HEAD</AllowedMethod>
```

Comment:
Does this also need `<AllowedMethod>DELETE</AllowedMethod>`? 
DELETE doesn't need the ETag related headers, just the `allow origin: *`. I don't recall if this config acted more like a blacklist or a whitelist